ee/tuf/autoupdate.go

```diff
@@ -528,6 +541,15 @@ func (ta *TufAutoupdater) checkForUpdate(binariesToCheck []autoupdatableBinary)
 	if updatedVersion, ok := updatesDownloaded[binaryLauncher]; ok {
 		// Only reload if we're not using a localdev path
 		if ta.knapsack.LocalDevelopmentPath() == "" {
+			ctx := context.Background()
```

Comment:
I'll defer to becca, but I'm not sure this is the right place

Reply:
Thinking...we have both `LauncherVersion` and `OsqueryVersion` inside the enrollment details, so that's why I'd wanted it updated after autoupdate. If we want to keep it here for that reason, then it probably makes more sense to _only_ update `EnrollmentDetails.LauncherVersion` and `EnrollmentDetails.OsqueryVersion` -- we don't need to regenerate the rest of the enrollment details at this time. What do you think?